src/demo-app/a11y/a11y.html

```diff
@@ -1,9 +1,12 @@
 <h1>Accessibility Demo</h1>
 
 <nav>
+  <a (click)="skipNavigation()" href="javascript:void(0)">Skip navigation</a>
```

Comment:
Chatted w/ Jason a bit about this to see if we could do any kind of workaround. And... not really.

Does it work if you make the `href="#maincontent"` and still use the click handler to do the work?